proto/storage/config.proto

```diff
@@ -55,6 +55,10 @@ message DecommissionedClusterRetentionConfig {
     google.protobuf.Timestamp created_at       = 4;
 }
 
+message ReportRetentionConfig {
+    int32 retention_duration_days              = 1;
```

Comment:
Will you add the sized-based retention in a separate PR?

Reply:
yes